### ISSUE TITLE
Add Support Docker Compose

### ISSUE THREAD

**Issue by YuraKril** [carlpett/teamcity-container-cloud#10]
Add support starting agents with the Docker Compose yml file.

**Comment by carlpett** [comment]
@YuraKril Thanks for the suggestion! I haven't used Compose for quite a while - could you elaborate a bit on the scenario you had in mind? Do you want to start linked containers along with the agent, or is there some other feature of Compose which gives it an advantage over using the Docker api?

**Comment by YuraKril** [comment]
Yes, I want to start linked containers along with the agent. Also I need to run several concurrent builds to a configuration in TeamCity.
